File: src/main/java/quizapi/model/Answer.java
package quizapi.model;

import javax.persistence.*;

@Entity
@Table(name = "answers")
public class Answer {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", unique = true, nullable = false)
    private Long id;

    @Column(name = "is_right", nullable = false)
    private Boolean isRight;

    @Column(name = "description", nullable = false)
    private String description;

    @ManyToOne(targetEntity = Question.class)
    private Question question;

    public Answer() {
    }

    public Answer(Boolean isRight, String description, Question question) {
        this.isRight = isRight;
        this.description = description;
        this.question = question;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Boolean getRight() {
        return isRight;
    }

    public void setRight(Boolean right) {
        isRight = right;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Question getQuestion() {
        return question;
    }

    public void setQuestion(Question question) {
        this.question = question;
    }
}

File: src/main/kotlin/com/shkila/quizapi/model/Answer.kt
package com.shkila.quizapi.model

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.FetchType
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
import javax.persistence.JoinColumn
import javax.persistence.ManyToOne
import javax.persistence.Table

@Entity
@Table(name = "answers")
data class Answer(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        @Column(name = "id", unique = true, nullable = false)
        val id: Long? = null,

        @Column(name = "is_right", nullable = false)
        var right: Boolean? = null,

        @Column(name = "description", nullable = false)
        var description: String? = null,

        @ManyToOne(fetch = FetchType.EAGER)
        @JoinColumn(name = "question_id", nullable = false)
        var question: Question? = null
)
